.gitignore

```diff
@@ -1,9 +1,27 @@
+# test code
 test-code/
-node_modules/
+
+# histories
+logs/
+socket/
 .idea/
-api-server/code-container/executables/
-api-server/deployed-apps/
+
+# deployments
+platform/api-server/code-container/executables/
+platform/api-server/deployed-apps/
```

Comment:
This needs to be removed, right?

Reply:
`platform/api-server/code-container/executables/` can be removed.
`platform/api-server/deployed-apps/` will be used for test app.